flink-connector/flink/src/test/java/org/apache/gravitino/flink/connector/iceberg/TestIcebergPropertiesConverter.java

```diff
@@ -39,7 +39,7 @@ void testCatalogPropertiesWithHiveBackend() {
                 "hive-uri",
                 IcebergPropertiesConstants.GRAVITINO_ICEBERG_CATALOG_WAREHOUSE,
                 "hive-warehouse",
-                "key1",
```

Comment:
`key1` and `value1` is used to check the key not in converted key map will not in the transformed properties. You could add `flink.bypass.key2` and `value2`

Reply:
fixed